src/Persistance/Json/JsonFileHandler.php

```diff
@@ -109,6 +109,25 @@ public function write(string $filename, mixed $jsonable): void
         }
     }
 
+    /**
+     * Remove the file from storage.
+     *
+     * @param string $filename
+     * @return void
+     */
+    public function unlink(string $filename): void
+    {
+        // Given the path to the file using the slug
+        $slug = $this->pathTo($filename);
```

Comment:
Suggestion: `$slug` is an inaccurate name for this variable, because the `pathTo()` method doesn't return the slug - it returns the full path to the file. Take a look at what the variable is called in the other methods to get an idea of what it could be called.

Reply:
I updated the variable name to $path.